styles/_blog-article.less

```diff
@@ -167,6 +167,7 @@
     img.arrow {
       width: 28px;
       height: 16px;
+      color: @badgerRedOnWhite;
```

Comment:
You have added a colour style to an image. We need to change the image to get a new colour for the arrow.